src/main/scala/scalismo/ui/model/properties/ScalingProperty.scala

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright (C) 2016  University of Basel, Graphics and Vision Research Group
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package scalismo.ui.model.properties
+
+object ScalingProperty {
```

Comment:
A thought: A lot of code seems to be a duplicate from the `RadiusProperty`. Would it make sense to have for PointCloudVisualization a standard radius of 1mm and a `ScalingProperty` instead? This would remove the `RadiusProperty` related classes then.